main.js

```diff
@@ -55,4 +55,4 @@ function updateTwitterIntent(quote) {
   var intentAddress = 'https://twitter.com/intent/tweet?text=';
```

Comment:
Going through the ES6 course you linked. Imagine my surprise to learn that there are different schools of thought on if/when/how to use `let`, `const` and `var`. Man, using `var` for everything was such a breeze! 

I'm leaning towards using only `let` and `const`: using `let` by default, unless I know for sure that the value of the variable will not be reassigned, and then possibly refactoring variables to `const` at the end if it makes sense. The mental overhead of yet _another_ decision to make feels very "heavy" right now.

My thinking is that only having two variables to choose between lowers the mental overhead, and defaulting to `let` when unsure relieves me of having to know for certain whether the variable will get reassigned.

What do you think?

Reply:
well, hard to say. When I declare a variable I usually already know if it will change or not. However, I can see that this might be difficult sometimes when you are exploring different solutions. I would still flip it around. use `const` as long as you can. if it crashes then go to `let`. reasoning: `let` always works and is weaker, `const` expresses an intention and is stronger.